source/speech/speech.py

```diff
@@ -1125,8 +1125,17 @@ def speakTextInfo(
 	)
 
 	speechGen = GeneratorWithReturn(speechGen)
+	symbolLevel = None

```

Comment:
```suggestion
	symbolLevel: characterProcessing.SYMLVL = None
```

Reply:
Awaiting answer to https://github.com/nvaccess/nvda/pull/11856#discussion_r637799407